### PR TITLE
Fix buggy drag and drop

### PR DESCRIPTION
**Pull request by KyrneDev**
The current Tags drag and drop in admin is really buggy and hard to use. It is almost impossible to use on mobile. This updates the sortable library to something more modern which makes it much, much easier to use (and has a nice little animation).

### Before: 
OldVersion

### Mobile (totally unusable):
Old Mobile

### New: 
NewVersion

### Mobile: 
New Mobile

### CODE REVIEW

**Comment by KyrneDev** [comment]
Please note that I changed to Dragula as per @datitisev's request. The functionality on mobile remains the same, but there is no animation as shown above.

**Comment by askvortsov1** [comment]
@KyrneDev @datitisev are there plans to make additional changes in animation to this, or is this ready for review?

**Comment by datitisev** [comment]
Imma play around with it.

**Comment by datitisev** [comment]
FYI - you removed the dist files instead of resetting them to a previous commit. That is fine, but you got me confused as to why the dist folder didn't exist.

File: js/src/admin/components/TagsPage.js [hunk 1]
```diff
@@ -80,18 +80,24 @@ export default class TagsPage extends Page {
...
   config() {
-    sortable(this.$('ol, ul'), {
-      acceptFrom: 'ol,ul'
-    }).forEach(this.onSortUpdate.bind(this));
+      this.$('.TagList').get().map(e => {
+          sortable.create(e, {
+              group: 'tags',
+              animation: 150,
+              swapThreshold: 0.65,
+			  dragClass: 'sortable-dragging',
```
Comment (datitisev) [inline js/src/admin/components/TagsPage.js:88]: Indent this correctly please 🙂

**Comment by datitisev** [comment]
Oh, one thing I forgot to add. We probably want to expose sortablejs to window - so other extensions can take advantage of it or lazy load their own.

**Comment by askvortsov1** [comment]
How would we do that? webpack/importing/exporting JS is still a minefield for me

**Comment by datitisev** [comment]
With `expose-loader`. It'd be like [this](https://github.com/flarum/core/blob/3b1f5ca07b879e1ccc2466bac8fbf343d2359e75/js/src/common/index.js#L3), so `import 'expose-loader?sortable!sortablejs`.

**Comment by clarkwinkelmann** [comment]
> @datitisev  We probably want to expose sortablejs to window - so other extensions can take advantage of it or lazy load their own

I'm not seeing a benefit here. If another extensions needs `sortable` but doesn't depend on the Tags extension, they will need to include the `sortable` library in their dist file anyway in case Tags isn't being used.

Unless we also have a way for extension to conditionally add javascript dependencies only when it's not already registered, there won't be any benefit from a bundle size perspective, and I'm not seeing any other possible benefit.

**Comment by datitisev** [comment]
@clarkwinkelmann I mentioned lazy-loading because they can still request a JS file after building - it'll just need to be in the extension's assets folder or use an external CDN.

See https://github.com/FriendsOfFlarum/drafts/blob/aece8dfe3f0b294f17807258b229f16857a0957a/js/src/forum/components/ScheduleDraftModal.js for an example.

**Pull request merged**